src/wwwroot/controllers/PlanCtrl.js

```diff
@@ -174,6 +184,42 @@
       }      
     }
 
+    function isUpgradeablePlan(limit){
+      //Check constraints to validate that plan is upgradeable
+
+      //Set to true all errors to prevent access if service is not working
+      var errorsPlanUpgrade = {'isDKIMError': true, "isDeliveredError": true, "isDeliveredAndDKIMError": true};
```

Comment:
There is already a global variable to check if there is a Dkim, or domain error to be configured, it's currently being used in the header, review the function called loadLimits in MainCtrl.

Reply:
Thanks @mvillaseco I will check it and fix up this 👍